.gitignore

```diff
@@ -130,3 +130,5 @@ compile_commands.json
 CTestTestfile.cmake
 
 # End of https://www.gitignore.io/api/c++,cmake,clion
+
+.vs/
```

Comment:
This shouldn't be needed anymore. I added a `.gitignore` for both `vs` and `vscode` in the PR that added build scripts.